apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -15,10 +15,15 @@ const (
 var (
 	m = image.Mirror
 
-	// k8sVersionsCurrent are the latest versions available for installation
-	k8sVersionsCurrent = []string{
-		"v1.9.7-rancher2-2",
-		"v1.10.12-rancher1-1",
+	K8sBadVersions = map[string]bool{
+		"v1.9.7-rancher1":    true,
+		"v1.10.1-rancher1":   true,
+		"v1.8.11-rancher1":   true,
+		"v1.8.10-rancher1-1": true,
+	}
+
+	// K8sVersionsCurrent are the latest versions available for installation
+	K8sVersionsCurrent = []string{
 		"v1.11.6-rancher1-1",
 		"v1.12.4-rancher1-1",
```

Comment:
can we replace 1.12.4 with 1.12.5? And make `v1.13.1-rancher1-1` a default version https://github.com/rancher/types/blob/da7f9038c5159b9551706d6ee8e081746471858d/apis/management.cattle.io/v3/k8s_defaults.go#L12

Reply:
latest is 1.13.1-rancher1-1 and 1.12 was updated to 1.12.5-rancher1-1